docs/blog/20/5-gettig_started_to_vitis_acceleration_flow_with_zynq_7000.md

```diff
@@ -0,0 +1,351 @@
+![zc706_sw11](img/5-zc706_sw11.png)
+
+### Creating SD card Image
+
+Insert SD card to your PC and check its name with command.
+
+```console
+$ sudo fdisk -l
+```
+
+Most likely, SD card will be named **mmcblk0** and it's partitions will be printed
+on console.
+
+```console
+$ sudo umount /dev/mmcblk0
+```
+
+After making sure it is the SD card you want to use, format and copy SD card image
+***sd_card.img*** created with `build.sh` script. Write it to your SD card.
+
+```console
+$ dd if=/dev/mmcblk0 of=./package/sd_card.img
+```
+
+After these operations, remove and insert SD card to your PC, you should see
+two partitions. One for root file system in ext4 format and the other one that includes
+`BOOT.bin`, `app.exe` and other related stuff.
+
+![5-sd_card](img/5-sd_card.png)
+
+Insert SD card to ZC706, power on the board and wait until it boots. Execute
+commands below on ZC706 petalinux terminal screen.
+
+```console
+$ mount /dev/mmcblk0p1 /mnt
+$ cd /mnt
+$ cp platform_desc.txt /etc/xocl.txt
+$ export XILINX_XRT=/usr
+$ export XILINX_VITIS=/mnt
+$ ./app.exe
+```
+
+![hw](img/5-hw.png)
+
+## Modifications on host.cpp
+
+I recommend you to change sections below in your `host.cpp` file for better
+understanding of vector addition example.
+
+```c++
+// ------------------------------------------------------------------------------------
+// Step 4: Check Results and Release Allocated Resources
+// ------------------------------------------------------------------------------------
+    bool match = true;
+    int miss_counter = 0;
+    for (int i = 0 ; i < DATA_SIZE ; i++){
+        int expected = in1[i]+in2[i];
+        if (out[i] != expected){
+            std::cout << "Error: Result mismatch" << std::endl;
+            std::cout << "i = " << i << " CPU result = " << expected << " Device result = " << out[i] << std::endl;
+            match = false;
+            miss_counter++;
+            //break;
+        }
+        std::cout << "i = " << i << " CPU result = " << expected << " Device result = " << out[i] << std::endl;
+    }
+    std::cout << miss_counter << " results mismatched" << std::endl;
+```
+
+You will not need to run scripts every time since you only changed `host.cpp`.
+Compile the `app.exe` again with the command below inside `hw` folder.
+
+```console
+$ arm-linux-gnueabihf-g++ -Wall -g -std=c++11 ../../src/host.cpp -o app.exe -I${SYSROOT}/usr/include/xrt -L${SYSROOT}/usr/lib -lOpenCL -lpthread -lrt -lstdc++ --sysroot=${SYSROOT}
+```
+
+Copy `app.exe` to SD card attached to your board, with ssh from your PC.
+
+```console
+$ scp app.exe  root@<board_ip_address>:/mnt
+```
+
+You can run `app.exe` from board's console again.
+
+```console
+$ ./app.exe
+```
+
+!!! Warning
+    Your board should have an IP address and should be connected to your PC
+    over ethernet for ssh connection.
+
+![after_change](img/5-after_change.png)
+
+<!---Refs --->
+[^1f]:https://github.com/Xilinx/Vitis-Tutorials/tree/master/Getting_Started/Vitis
```

Comment:
Önceki yazılarda da burada eksik bırakmışız. Böyle olunca linkler tıklanmıyor, eskileri için de issue açacağım. Bunları da yazı içindeki linkler gibi `<>` yapabilir misin? `:` dan sonra da bir boşluk atabilirsin.